### PR TITLE
Update regular check guide

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR updates the regular check guide.
It describes how to check Kubernetes resources, Prometheus alerts, and Grafana metrics.
Please take a look!

### CODE REVIEW

**Comment by feeblefakie** [comment]
@superbrothers Any recommendations for this?

File: docs/RegularCheck.md [hunk 1]
```diff
@@ -1,113 +1,73 @@
...
-* `logging` for Fluent bit
-* `monitoring` for Prometheus
+* `default` (or specified namespace when you deploy Scalar products) for the Scalar product deployment
+* `monitoring` for the Prometheus Operator and Loki
 
 What to check:
 
-* `STATUS` is all `Running`
-* `RESTARTS` number is all 0
+* `STATUS` is all `Running` (ScalarDL Schema Loader is `Completed`)
```
Comment (feeblefakie) [inline docs/RegularCheck.md:16]: (ScalarDL Schema Loader is `Completed`) should be removed or modified since the following shows the information about scalardb pods?

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
Left one minor comment, so please fix it as needed.

File: docs/RegularCheck.md (hunk 1 above)
Comment (kota2and3kan) [reply docs/RegularCheck.md]: @feeblefakie  
Thank you for your review!

I only described ScalarDB's example since the pod name is not important point in this section (the important thing is pod status) and it is redundant if we describe all examples of ScalarDB, Ledger, and Auditor (basically only the pod names are different).

And, I covered the ScalarDL Schema Loader case with this note, but it is a bit confusing.
Also, this document explains what we should check on a **regular basis**.
However, we only need to check the status of the ScalarDL Schema Loader once when we deploy it.
In other words, we don't need to check the status of the Schema Loader on a regular basis.

So, I removed this note.

**Review by superbrothers — APPROVED** [review]
